src/user/user.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { UserRepository } from './user.repository';
+import { InjectRepository } from '@nestjs/typeorm';
+
+
+@Injectable()
+export class UsersService {
+        constructor(
+          @InjectRepository(UserRepository) private usersRepository: UserRepository,
+        ) {}
+      
+        // findAll(): Promise<User[]> {
```

Comment:
why are these comments here?

Reply:
I was testing something and  i guess i forgot to remove them 
ill do remove them now